demo/inferno-router-demo/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "inferno-router-demo",
+  "version": "8.0.5",
+  "description": "Influence CMS Demo",
+  "author": "Sebastian Ware <sebastian@urbantalk.se> (https://github.com/jhsware)",
+  "license": "SEE LICENSE IN LICENSE",
+  "source": "src/index.html",
+  "browserslist": "> 0.5%, last 2 versions, not dead",
+  "scripts": {
+    "dev": "npm-run-all -l --parallel  dev:*",
+    "dev:frontend": "FORCE_COLOR=1 parcel",
```

Comment:
these scripts does not run on windows due to different syntax of environment variables than unix, please use cross-env

Reply:
WILL FIX

Reply:
Fixed in 65c7d77a